### DISCUSSION TITLE
s3box and TTS voices.

### DISCUSSION THREAD

**Discussion by rofo69** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#63] [General]
If I ask the s3 to assist, it replies in a male voice, which if I understand it correctly, is defined by the TTS pipeline in HA, which in my case is 'alan (medium)':-

![image](https://github.com/user-attachments/assets/be79d1ae-0c07-4f59-9efd-62101c8f9f85)

If I set an response reply in an automation, it replies with the same voice.

If I simply send a TTS to the s3 media player using piper, I assume its using the voice defined here:-

![image](https://github.com/user-attachments/assets/97540ed1-dcb4-4954-9adb-f0e5ffda7517)

As its not using the pipeline configuration, I'm just sending TTS audio to the media player.

However, when configuring piper as a stand alone TTS, the list of voices is different to those I can select from the assist pipeline, and 'alan' is no longer available as a choice:-

![image](https://github.com/user-attachments/assets/f1180073-8ddf-435a-9497-9052c42c44dc)

This means, depending on how I send TTS audio to the s3, I get one of two different voices. Is there any way to unify these ?

**Comment by rhyswaywood** [comment]
Scroll up the list a bit. It should be under en_GB-alan-medium (not en-US)

Your TTS pipeline is set to en_GB as the language.

![image](https://github.com/user-attachments/assets/aa8594ac-bfd9-42b3-bc41-1d98b933f428)

**Reply by rofo69** [reply]
Really not sure how I missed that! Cheers
